src/utils.ts

```diff
@@ -115,6 +115,7 @@ export function executeWithFreshModules<T>(f: () => T): T {
       // Ensure every page gets the same patched 'next/document' instance
       // imported before tests by "src/testHelpers.ts"
       jest.mock('next/document', () => jest.requireActual('next/document'));
+      jest.mock('react', () => jest.requireActual('react'));
```

Comment:
Any ideas why this was not needed before and started popping up now? Anyhow I think we are safer if we have it explicitly required.

Reply:
I don't know why it popped up just now, but I felt we'd have been forced to do it sooner or later.

I'm realising that since we rely on this, users should never reset their modules, right?